website/docs/development/contributing.md

```diff
@@ -26,7 +26,7 @@ conda activate hydra37
 ```
 From the source tree, install Hydra in development mode with the following command:
 ```
-pip install -e .[dev] -e .
+pip install -e '.[dev]' -e .
```

Comment:
I added the single quotes to properly install it for _both_ Bash and Zsh (it fails because of the `[]`).

Reply:
roger.